tests/README.md

```diff
@@ -40,6 +38,41 @@ To run autest manually, the recommended way is to follow these steps:
 3. **cd gold_tests**: enter the directory containing the test files.
 4. **autest --ats-bin user_ats_bin**: run autest where user_ats_bin is the bin directory in the user's ats directory.
 
+# Running tests
```

Comment:
You might want to add a note to configure the trafficserver build to install in a location as such ~/ats

I tend to find this helps a lot vs installing on the system itself